### ISSUE TITLE
[5.4] Translator Error

### ISSUE THREAD

**Issue by computerfr33k** [flugger/laravel-responder#31]
I am getting an exception thrown after I upgraded to 5.4.

```php
  [Symfony\Component\Debug\Exception\FatalThrowableError]                                                                                                     
  Type error: Argument 2 passed to Flugg\Responder\Http\ErrorResponseBuilder::__construct() must be an instance of Symfony\Component\Translation\TranslatorI  
  nterface, instance of Illuminate\Translation\Translator given, called in vendor/flugger/laravel-responder/src  
  /ResponderServiceProvider.php on line 130 
```

**Comment by computerfr33k** [comment]
Looks like Laravel 5.4's Translator is now implementing Illuminate\Contracts\Translation\Translator interface instead of Symfony's Translator interface.

**Comment by flugger** [comment]
Hey! Thanks for reporting, should be fixed from tag `1.2.19`. Lemme know if you encounter any other problems
